html5/js/Window.js

```diff
@@ -1544,27 +1538,23 @@ class XpraWindow {
           paint_bitmap();
           return;
         }
-        const j = new Image();
-        j.onload = () => {
-          if (j.width == 0 || j.height == 0) {
-            paint_error("invalid image size: " + j.width + "x" + j.height);
+        const index = new Image();
+        index.addEventListener("load", () => {
+          if (index.width == 0 || index.height == 0) {
+            paint_error(`invalid image size: ${index.width}x${index.height}`);
```

Comment:
`index` is definitely the wrong name here, should be `image` is that name is free - or just keep `j` and keep the history intact.

Reply:
Good catch, fixed!